src/app/technical-record/technical-record.component.html

```diff
@@ -413,7 +421,11 @@ <h1 class="govuk-heading-xl title">Technical record</h1>
         <div
           class="auto-pad"
           [@.disabled]="true"
-          *ngIf="activeRecord.vehicleType === 'trl' || activeRecord.vehicleType === 'hgv' || activeRecord.vehicleType === 'psv'"
+          *ngIf="
+            activeRecord.vehicleType === 'trl' ||
+            activeRecord.vehicleType === 'hgv' ||
+            activeRecord.vehicleType === 'psv'
+          "
```

Comment:
do want to change this to `techRecHelpers.isStandardVehicle(activeRecord.vehicleType`